src/Microsoft.DotNet.Build.CloudTestTasks/UploadClient.cs

```diff
@@ -114,7 +113,11 @@ public string EncodeBlockIds(int numberOfBlocks, int lengthOfId)
                                 req.Content = contentStream;
                                 using (HttpResponseMessage response = await client.SendAsync(req, ct))
                                 {
-                                    this.log.LogMessage(
+                                    if (!response.IsSuccessStatusCode)
```

Comment:
What are we throwing exceptions here now? Who do you expect to catch/handle/report the errors?